doc/classes/Script.xml

```diff
@@ -43,6 +43,84 @@
 				Returns a dictionary containing constant names and their values.
 			</description>
 		</method>
+		<method name="get_script_documentation_list">
+			<return type="Array" />
+			<description>
+				Returns an array of dictionary containing the documentation of this script and its subclasses.
+				Each dictionary contains the following structure:
+				[codeblocks]
+				[gdscript]
```

Comment:
For a code block you can just use `[codeblock]`, there is no need to use tabbed code blocks here.

That said, this is probably too excessive for a method description. It's worth to have, but I'm not sure where best to put it.

Reply:
Modified, thanks for the reminder